dist/tokens/ts/core-light-colors.d.ts

```diff
@@ -10,4 +10,4 @@ export const purple : {100 : string,200 : string,300 : string,400 : string,500 :
 export const orange : {100 : string,200 : string,300 : string,400 : string,500 : string,600 : string,700 : string,800 : string};
 export const grey : {100 : string,200 : string,300 : string,400 : string,500 : string,600 : string,700 : string,800 : string};
 export const shades : {white : string,black : string,transparent : string};
-export const yellow : {100 : string,200 : string,300 : string,400 : string,500 : string,600 : string,700 : string,800 : string};
\ No newline at end of file
+export const yellow : {100 : string,200 : string,300 : string,400 : string,500 : string,600 : string,700 : string,800 : string};export const boxShadows: {};
```

Comment:
hmm i think we have an issue with core all around?

Reply:
I forgot to mention I noticed this yesterday! We don't have any box shadow tokens for core so it's empty. I'll add a conditional within the formatter or look for a similar solution

Reply:
makes sense!! can do that in a follow up!